### PR TITLE
Hany branch - Pull Request

### PR DESCRIPTION
**Pull request by haniscape**
Dear Mladen please review my branch modifications and close my PR as required.
Thanks very much

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,61 +1,224 @@
...
+
+.book .title {
+  font-size: 1.1em;
+  font-weight: 700;
+  margin-bottom: 10px;
+}
+
+.book .author {
+  font-size: 1.1em;
+  font-weight: 400;
```
Comment (mladenuzelac) [inline css/style.css:100]: author's name should be italic

File: css/style.css [hunk 2]
```diff
@@ -1,61 +1,224 @@
...
+}
+
```
Comment (mladenuzelac) [inline css/style.css]: We never mentioned that responsiveness should be a part of the task.
You may delete this part.

File: css/style.css (hunk 1 above)
Comment (haniscape) [reply css/style.css:100]: I did that in the class author

File: css/style.css (hunk 2 above)
Comment (haniscape) [reply css/style.css]: Ok I reverted back to Desktop only design , I have a backup of the media queries in case we needed that later on.

**Review by mladenuzelac — APPROVED** [review]
Changes are done

**Pull request merged**